setup.py

```diff
@@ -22,17 +22,30 @@ def write_version_py(filename=None):
         fh.close()
 
 
-write_version_py()
+cfd = os.path.join(
```

Comment:
You probably do not need to do this. The `version.py` is necessary for the package and it written during setup so that the version of the package is controlled by the setup.py.